sabnzbd/par2file.py

```diff
@@ -106,40 +109,63 @@ def parse_par2_file(fname: str, md5of16k: Dict[bytes, str]) -> Tuple[str, Dict[s
     http://parchive.sourceforge.net/docs/specifications/parity-volume-spec/article-spec.html
     """
     total_size = os.path.getsize(fname)
+    metadata = {}
+    metadata["nr_files"] = -1
+    filedata = {}
```

Comment:
Could you make `metadata` and `filedata` into `dataclass`? It's easier to read than dict's.

Reply:
Sorry, I don't see how to get rid of the filedata dict? How to connect the CRC32 values to the right file?